src/CoreGraphics/CGDisplay.cs

```diff
@@ -0,0 +1,171 @@
+		public static extern nint GetTypeID ();
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern CGRect CGDisplayBounds (uint display);
+
+		public static CGRect GetBounds (int display)
+		{
+			return CGDisplayBounds ((uint)display);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern nuint CGDisplayPixelsWide (uint display);
+
+		public static nint GetWidth (int display)
+		{
+			return (nint)CGDisplayPixelsWide ((uint)display);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern nuint CGDisplayPixelsHigh (uint display);
+
+		public static nint GetHeight (int display)
+		{
+			return (nint)CGDisplayPixelsHigh ((uint)display);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern int CGSetDisplayTransferByFormula (uint display, float redMin, float redMax, float redGamma, float greenMin, float greenMax, float greenGamma, float blueMin, float blueMax, float blueGamma);
+
+		public static int SetDisplayTransfer (int display, float redMin, float redMax, float redGamma, float greenMin, float greenMax, float greenGamma, float blueMin, float blueMax, float blueGamma)
+		{
+			return CGSetDisplayTransferByFormula ((uint)display, redMin, redMax, redGamma, greenMin, greenMax, greenGamma, blueMin, blueMax, blueGamma);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern uint CGDisplayGammaTableCapacity (uint display);
+
+		public static int GetGammaTableCapacity (int display)
+		{
+			return (int)CGDisplayGammaTableCapacity ((uint)display);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary, EntryPoint = "CGDisplayRestoreColorSyncSettings")]
+		public static extern void RestoreColorSyncSettings ();
+
+		[Deprecated (PlatformName.MacOSX, 10, 9)]
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern bool CGDisplayIsCaptured (uint display);
+
+		public static bool IsCaptured (int display)
+		{
+			return CGDisplayIsCaptured ((uint)display);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern int CGDisplayCapture (uint display);
+
+		public static int Capture (int display)
+		{
+			return CGDisplayCapture ((uint)display);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern int CGDisplayCaptureWithOptions (uint display, uint options);
+
+		public static int Capture (int display, int options)
+		{
+			return CGDisplayCaptureWithOptions ((uint)display, (uint)options);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern int CGDisplayRelease (uint display);
+
+		public static int Release (int display)
+		{
+			return CGDisplayRelease ((uint)display);
+		}
+
+		[DllImport (Constants.CoreGraphicsLibrary, EntryPoint = "CGCaptureAllDisplays")]
+		public static extern int CaptureAllDisplays ();
+
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern int CGCaptureAllDisplaysWithOptions (uint options);
+
+		public static int CaptureAllDisplays (int options)
```

Comment:
That `options` should be an enum (not a `int` or `uint`), see https://developer.apple.com/reference/coregraphics/cgcaptureoptions?language=objc